src/store/index.js

```diff
@@ -186,12 +188,21 @@ export default new Vuex.Store({
                 return tile.type === type && tile.unitId === unit.id
             })
         },
+        unitOnType: (state, getters) => (type) => {
```

Comment:
We should probably have `allUnitsOnType` call this instead of duplicating the code in that getter.

Reply:
I don't think allUnitsOnType would work in this situation since it checks to see if all units are on their designated type tiles- whereas the timer tile needs to only check if there is one/some units a time tile in order to switch the timer flip button on. Also the timer tile isn't concerned with the unit on it so the unit id check that allUnitsOnType does isn't necessary

Reply:
I was originally thinking that `allUnitsOnType` would call `unitOnType` in a loop, but I now see the `unitId` check is incompatible. I may rename `allUnitsOnType` to something more descriptive like `allUnitsOnMatchingType` or something....